bls/src/main/java/tech/pegasys/teku/bls/BLS.java

```diff
@@ -91,8 +109,10 @@ public static boolean verify(BLSPublicKey publicKey, Bytes message, BLSSignature
    *
    * @param signatures the list of signatures to be aggregated
    * @return the aggregated signature
+   * @throws IllegalArgumentException if any of supplied signatures is invalid
    */
-  public static BLSSignature aggregate(List<BLSSignature> signatures) {
+  public static BLSSignature aggregateSignatures(List<BLSSignature> signatures)
```

Comment:
As per my comment on `aggregatePublicKeys` - if that is moved elsewhere, then this can be reverted to the same name it has in the standard (i.e. `aggregate()` - modulo a captial letter).

Reply:
Commented on `aggregatePublicKeys`